aztec/src/main/kotlin/org/wordpress/aztec/AztecText.kt

```diff
@@ -888,4 +895,81 @@ class AztecText : EditText, TextWatcher {
         }
     }
 
+    fun insertMedia(drawable: Drawable?, attributes: Attributes) {
+        lineBlockFormatter.insertMedia(drawable, attributes, onMediaTappedListener)
+    }
+
+    fun removeMedia(attributePredicate: AttributePredicate) {
+        text.getSpans(0, 999999999, AztecMediaSpan::class.java).forEach {
```

Comment:
Ah, no special reason other than one less method call and laziness that stuck. Changed with d057ca8!